masonry/src/widget/tests/safety_rails.rs

```diff
@@ -53,6 +53,10 @@ fn check_forget_to_recurse_text_event() {
 
 #[should_panic(expected = "not added in method lifecycle")]
 #[test]
+#[cfg_attr(
+    not(debug_assertions),
+    ignore = "This test doesn't work without debug assertions (i.e. in release mode). See https://github.com/linebender/xilem/issues/477"
+)]
```

Comment:
I don't think you need to link to the issue. Doing a `cfg_attr` fixes it.

For the message I'd write something like:

"Safety rails are disabled when debug_assertions is false (i.e. in release mode)."